File: Cpp/Project Euler #53.cpp
#include <iostream>

double fact(int);

int main()
{
	int count = 0;
	for (int n = 23; n < 101; n++)
	{
		for (int r = 1; r <= n; r++)
			if (fact(n) / (fact(r) * fact(n - r)) > 1000000)
				count++;
	}
	std::cout << count;
	return 0;
}

double fact(int num)
{
	if (num == 0)
		return 1;
	double prod = num;
	for (int i = 2; i < num; i++)
		prod *= i;
	return prod;
}

File: Python/Project Euler #53.py
def fact(num):
    if num == 0:
        return 1
    prod = num
    for i in range(2, num):
        prod *= i
    return prod
def select(n, r):
    return fact(n) / (fact(r) * fact(n - r))
count = 0
for i in range(23, 101):
    for j in range(1, i + 1):
        if select(i, j) > 1000000:
            count += 1
print(count)
